cluster/saltbase/salt/kube-proxy/kube-proxy.manifest

```diff
@@ -56,7 +56,7 @@ spec:
   hostNetwork: true
   initContainers:
   - name: touch-lock
-    image: busybox
```

Comment:
`gcr.io/google_containers/busybox:1.24` is pretty old (much older than `:latest` in dockerhub) - why are we using it?

Reply:
Updated to `gcr.io/google_containers/busybox:1.27`, corresponding to `busybox:1.27`.